src/components/structures/UserMenu.tsx

```diff
@@ -33,6 +33,15 @@ import Modal from "../../Modal";
 import LogoutDialog from "../views/dialogs/LogoutDialog";
 import SettingsStore from "../../settings/SettingsStore";
 import { findHighContrastTheme, getCustomTheme, isHighContrastTheme } from "../../theme";
+import {
+    RovingAccessibleButton,
+    RovingAccessibleTooltipButton,
+    RovingTabIndexContext,
+    findSiblingElement,
+    useRovingTabIndex,
+    Type
+} from "../../accessibility/RovingTabIndex";
```

Comment:
Previously, the `UserMenu` was half-navigable where the `input` would trap your focus and the dark mode toggle wasn't accessible.

Now you `Arrow` key through the options or up to the custom status input which you can `Tab` out of back to the options or `Shift + Tab` up to the dark mode toggle